### PR TITLE
Added documentation for Subscribe to file events

### PR DESCRIPTION
**Pull request by crsndr**
# Summary

---

**Related issue :**

### CODE REVIEW

File: content/en/users/data/management/datahub/api/_index.md [hunk 1]
```diff
@@ -218,60 +222,82 @@ The following variables should be set:
...
+$ curl -N -H "X-Auth-Token: $TOKEN" \
+    -X POST "https://$HOST/api/v3/oneprovider/changes/metadata/$SPACE_ID" \
+    -H "Content-Type: application/json" -d "@./changes_req.json"
+```
+
+This requires the permission set as following:
+
+![Viewing file popularity for smart caching](datahub-space-permissions.png)
+
+For groups or single users. For single users, one way to add one is to select
```
Comment (andrea-manzi) [inline content/en/users/data/management/datahub/api/_index.md:297]: what we are doing so far is a kind of workaround..i need to check with onedata guys what is the best way to achieve this

Comment (crsndr) [reply content/en/users/data/management/datahub/api/_index.md:297]: Shall  we keep it anyway for now as it has already been used twice and at the moment is not really documented anywhere. If a better way comes out maybe we can then update this.

Comment (andrea-manzi) [reply content/en/users/data/management/datahub/api/_index.md:297]: ok let's do as you suggest, and update it with new instructions when available

**Comment by andrea-manzi** [comment]
i think the image that you added is too big, see https://docs.egi.eu/documentation/503/users/data/management/datahub/api/#subscribe-to-file-events

**Comment by andrea-manzi** [comment]
> > i think the image that you added is too big, see https://docs.egi.eu/documentation/503/users/data/management/datahub/api/#subscribe-to-file-events
> 
> I've just pushed a version with a smaller image, let me know if it's OK once it will have been deployed.

@gwarf thanks a lot! looks good now

**Review by andrea-manzi — APPROVED** [review]
thanks!

**Pull request merged**